allauth/mfa/tests/test_views.py

```diff
@@ -71,7 +73,9 @@ def test_activate_totp_success(
                 {
                     "code": "123",
                 },
+                **{"HTTP_USER_AGENT": "test"},
             )
+    assert len(mail.outbox) == 1
```

Comment:
Would be good to add an assert here, something like: `assert mailoutbox[0].subject == "..."`.

Reply:
added subject assert